maintainers/maintainer-list.nix

```diff
@@ -2423,6 +2423,11 @@
     githubId = 185443;
     name = "Alexey Lebedeff";
   };
+  binarycat = {
+      email = "binarycat@envs.net";
+      github = "lolbinarycat";
```

Comment:
Please also add your `githubId` here. See the top of the file for instructions.

Reply:
and some form of `name`

Reply:
i didn't add it because it was described as optional, but i fixed it now.

Reply:
Understandable. As I see it, it is optional together with the `github` field, so *if* you set `github` you also need to set `githubId`.

Reply:
@piegamesde is correct, if you set github, you also have to set githubId so some of our automation, like inviting you in the github org works. I would recommend you to set the github stuff but it's not mandatory.

Reply:
Like I said, I added the `githubId` field.